src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -46,6 +46,7 @@
     <aura:attribute name="openOpportunities" type="List" description="List of open Opportunities" access="public" />
     <aura:attribute name="unpaidPayments" type="List" description="List of unpaid Payments" access="public" />
     <aura:attribute name="selectedDonation" type="SObject" description="Stored selected open donation. Can also be null or empty." access="public" />
+    <aura:attribute name="checkedValue" type="Boolean" description="Stores the checkbox value to be saved by an inputFields" />
```

Comment:
If I'm reading this correctly, this would fail if there where multiple checkbox fields - they all be overwriting values into this single attribute?